.travis.yml

```diff
@@ -25,10 +95,15 @@ install:
   - sudo apt-get install -y python-lxml
 before_script:
   - export CFLAGS="-g -O2 -Werror=pointer-arith -Werror=implicit-function-declaration"
+  - if [ "$TRAVIS_COMPILER" = "clang" ]; then
+      CFLAGS="$CFLAGS -std=c++11";
+    fi
```

Comment:
don't do that, `configure` is responsible for this kind of stuff, and it should be doing it alright.  And if it ain't, it might be worth fixing it.

Reply:
BTW: `CFLAGS` is for *C* flags, not C++ ones.  `CXXFLAGS` is for C++ flags.  So it makes no sense to set `CFLAGS=-std=c++11`, as it's basically saying "use C++11 when compiling C code".